js/renovation/ui/scroll_view/scrollable_utils.ts

```diff
@@ -12,8 +12,10 @@ export const DIRECTION_BOTH = 'both';
 export const SCROLLABLE_CONTENT_CLASS = 'dx-scrollable-content';
 export const SCROLLABLE_WRAPPER_CLASS = 'dx-scrollable-wrapper';
 export const SCROLLABLE_CONTAINER_CLASS = 'dx-scrollable-container';
+export const SCROLLVIEW_CONTENT_CLASS = 'dx-scrollview-content';
 export const SCROLLVIEW_TOP_POCKET_CLASS = 'dx-scrollview-top-pocket';
 export const SCROLLVIEW_BOTTOM_POCKET_CLASS = 'dx-scrollview-bottom-pocket';
+export const SCROLLABLE_DISABLED_CLASS = 'dx-scrollable-wrapper';
```

Comment:
Seems we need change className to dx-scrollable-disabled

Reply:
Fixed